server/index.js

```diff
@@ -169,18 +179,20 @@ module.exports = (_config = {}) => {
     );
     app.get('/server', (req, res) => res.status(200).send({ bcoinUri: uri }));
 
-    if (nodeClient) {
-      logger.info(`Connecting with ${clientConfig.str('client-id')} client`);
-      app.use('/bcoin', bcoinRouter(nodeClient));
-    }
+    app.use('/clients', clientRoutes(clients));
 
-    if (walletClient) {
-      app.use('/bwallet', bcoinRouter(walletClient));
-    }
+    // redirects to support old routes
+    app.use('/bcoin', (req, res) =>
+      res.redirect(307, `/clients/${clientConfig.str('id')}/node${req.path}`)
```

Comment:
I think that we should save `clientConfig.str('id')` in a global variable or maybe even better, have it attached to `req` so that we don't have to pull it from the config each request